levels/inside_perimeter/maps/lovelacehallway.json

```diff
@@ -0,0 +1,576 @@
, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 101, 12599, 12681, 12687, 12717, 12717, 12717, 12717, 12687, 12679, 11978, 11978, 11978, 11978, 12059, 11137, 11137, 11137, 11137, 11137, 12057, 11978, 11978, 11978, 11978, 11978, 12680, 12601, 11057, 11057, 101, 12639, 12684, 12727, 12757, 12757, 12757, 12757, 12727, 12719, 12717, 12717, 12717, 12720, 12721, 11509, 11506, 11507, 11508, 11509, 12719, 12720, 12717, 12717, 12717, 12720, 12686, 12641, 11057, 11057, 101, 12639, 12724, 12767, 12797, 12797, 12797, 12797, 12767, 12759, 12757, 11558, 12757, 12760, 12761, 11549, 11546, 11547, 11548, 11549, 12759, 12760, 12757, 11558, 12757, 12760, 12726, 12641, 11057, 11057, 101, 12639, 12764, 11137, 11137, 11137, 11137, 11137, 11137, 12799, 12797, 11598, 12797, 12800, 12801, 11137, 11137, 11137, 11137, 11137, 12799, 12800, 12797, 11598, 12797, 12800, 12766, 12641, 11057, 11057, 101, 12639, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 12641, 11057, 11057, 101, 12639, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 12641, 11057, 11057, 101, 12639, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 11137, 12641, 11057, 11057, 101, 12679, 12601, 11137, 11137, 11137, 11137, 11137, 11137, 12599, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 12681, 11057, 11057, 101, 12991, 12679, 12680, 12680, 12680, 12680, 12680, 12680, 12681, 12994, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12992, 12993, 11057, 11057, 101, 13041, 12991, 12992, 12992, 12992, 12992, 12992, 12992, 12993, 13044, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13043, 11057, 11057, 101, 13041, 13041, 13042, 13042, 13042, 13042, 13042, 13042, 13043, 13044, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13042, 13043, 11057, 11057, 101, 13091, 13041, 13042, 13042, 13042, 13042, 13042, 13042, 13043, 13094, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13092, 13093, 11057, 11057, 101, 101, 13091, 13092, 13092, 13092, 13092, 13092, 13092, 13093, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 11057, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 11057, 11057, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 101, 11057, 11057],
```

Comment:
It looks like you may have had the `Tile Layer Format` set to `CSV` in Tiled when saving this. We'll want to change it to `Base64 (uncompressed)` and save it again to avoid any bugs when loading the map (I've left a screenshot below showing how to change it, just in case)!

<img width="544" alt="tile-layer-format-tiled-setting-example" src="https://user-images.githubusercontent.com/51891409/202033119-00c01f8b-5557-4378-ac4e-01d6380bf321.png">

Reply:
Huh, wonder how that happened! I'll try to change that